tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "target": "esnext",
     "downlevelIteration": true,
     "esModuleInterop": true,
-    "module": "es2015",
+    "module": "esnext",
```

Comment:
Do we want to keep the changes in tsconfig.json?

Reply:
Nope, that's only for the `import.meta`